src/v3/components/time_picek/time_picker.style.ts

```diff
@@ -0,0 +1,97 @@
+export const StyleTimePickerPopper = {
+  '.MuiClockPointer-root': {
+    backgroundColor: 'var(--accent-main)',
+  },
+  '.MuiClock-pin': {
+    backgroundColor: 'var(--accent-main)',
+  },
+  '.MuiClockPointer-thumb': {
+    backgroundColor: 'var(--accent-main)',
+    borderColor: 'var(--accent-main)',
+  },
+  '.MuiClockNumber-root': {
+    color: 'var(--black)',
+    '&.Mui-selected': {
+      color: 'var(--always-white)',
+    },
+  },
+  '.MuiClock-clock': {
+    backgroundColor: 'var(--accent-200)',
+    height: '225px',
+    width: '225px',
+    padding: '2px 0 0 2px',
+    transform: 'scale(1.1)',
+  },
+  button: {
+    '&[title="PM"]': {
+      display: 'none',
+    },
+    '&[title="open previous view"]': {
+      display: 'none',
+    },
+    '&[title="open next view"]': {
+      display: 'none',
+    },
+    '&[title="AM"]': {
+      display: 'none',
+    },
+  },
+};
+
+export const StyleTimePickerPaper = {
+  sx: {
+    backgroundColor: 'var(--white)',
+    borderRadius: 'var(--radius-xxl)',
+  },
+};
+export const StyleTimePickerToolbar = {
```

Comment:
@eloyEntony: looking at some of the styles assigned for this constant, will it be possible to use the existing class defined in the global.css file, instead of adding them again one by one here?

Reply:
maybe we can create ts style (styles obj)?

Reply:
I see. We can leave it as it for now. Later if there is another great idea, we can rethink about it again.